src/execution/physical_plan/plan_set_operation.cpp

```diff
@@ -36,7 +36,7 @@ unique_ptr<PhysicalOperator> PhysicalPlanGenerator::CreatePlan(LogicalSetOperati
 		// EXCEPT is ANTI join
 		// INTERSECT is SEMI join
 		PerfectHashJoinStats join_stats; // used in inner joins only
-		JoinType join_type = op.type == LogicalOperatorType::LOGICAL_EXCEPT ? JoinType::ANTI : JoinType::SEMI;
+		JoinType join_type = op.type == LogicalOperatorType::LOGICAL_EXCEPT ? JoinType::LEFT_ANTI : JoinType::LEFT_SEMI;
```

Comment:
It seems like we're keeping `LogicalSetOperation` all the way until physical planning. Because of this, we can't flip left/right, and we can't do join ordering (we can't do that anyway because we can't reorder SEMI/ANTI yet, but when we can, it won't reorder these).

Not something for this PR, but this should be fixed by converting EXCEPT/INTERSECT to joins before entering the join order optimizer.

Reply:
hmm, yea. I filed an issue for it
https://github.com/duckdblabs/duckdb-internal/issues/568